source/features/remove-projects-tab.tsx

```diff
@@ -77,5 +59,20 @@ features.add({
 		pageDetect.isUserProfile,
 		pageDetect.isOrganizationProfile
 	],
-	init
+	exclude: [
+		pageDetect.canUserEditRepo,
+		pageDetect.canUserEditOrganization
+	],
+	init: removeProjectsTab
+}, {
+	include: [
+		pageDetect.isRepo,
+		pageDetect.isOrganizationProfile
+	],
+	exclude: [
+		() => !pageDetect.canUserEditRepo(),
+		() => !pageDetect.canUserEditOrganization()
```

Comment:
The only time the new projects should be added is if they have access? 
Am I wrong?

Reply:
Check

Reply:
I did. Just double checking that I am not missing something.